src/CMakeLists.txt

```diff
@@ -207,10 +207,14 @@ endif()
 if(UNIX AND NOT APPLE)
     set(keepassx_SOURCES
             ${keepassx_SOURCES}
-            quickunlock/Polkit.cpp
-            quickunlock/PolkitDbusTypes.cpp
             gui/osutils/nixutils/ScreenLockListenerDBus.cpp
             gui/osutils/nixutils/NixUtils.cpp)
+    if("${CMAKE_SYSTEM}" MATCHES "Linux")
```

Comment:
I suggest sticking to `if(LINUX)` as done in previous versions of this CMake code.

It is consistent with `if(... UNIX ... APPLE ... HAIKU ...)` and that one
ne occurence in src/CMakeLists.txt currently using `if("${CMAKE_SYSTEM}" STREQUAL "Linux")` slipped in with some macOS change, probably by accident, replacing the previous `if(LINUX)`.

Reply:
`if(LINUX)` didn't work on my system but that did, and im unsure as to why

Reply:
I can't find reference to `if(LINUX)` being valid anywhere in the [CMake docs](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Write-Platform-Checks#platform-variables).